applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -431,6 +444,19 @@ static void timeout_handler(struct k_work *work)
 	cancel_operation();
 }
 
+static void init_bt_stack_id_lut(void)
+{
+	if (IS_ENABLED(CONFIG_BT_PERIPHERAL)) {
+		for (size_t i = 0; i < ARRAY_SIZE(bt_stack_id_lut); i++) {
+			/* BT_ID_DEFAULT cannot be resetted - cannot be used */
+			bt_stack_id_lut[i] = i + 1;
+		}
+
+		/* Temporary fix for tests */
```

Comment:
?

Reply:
please do not introduce temporary fixes at this stage of a project

Reply:
It was introduced some time ago (required by automatic tests).
I will make this feature based on a Kconfig option and use it only in debug configurations (in separate PR).

Reply:
As I said before, I will make this feature depend on a Kconfig option and use it only in debug configurations (this will be added in separate PR).